omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/OrderResource2_2.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.openmrs.Order;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_10.OrderResource1_10;
+
+@Resource(name = RestConstants.VERSION_1 + "/order", supportedClass = Order.class, supportedOpenmrsVersions = { "2.2.*" }, order = 1)
```

Comment:
Does the order attribute add anything here? I doubt

Reply:
Strange, there must be something not done correctly, I know various resources with the same name and no order attribute and they work e.g `LocationResource1_8`, `LocationResource1_9` and `LocationResource2_0`

Changing to v2 IS NOT an option, as I mentioned earlier, it stands for the entire REST API version and that isn't about to change anytime soon.

Reply:
@djazayeri do you know how the order attribute is used on a resource?

Reply:
@fruether my guess is that you have both OrderResource_1_10 and OrderResource_2_2 both supporting OpenMRS version 2.2.x, I think you need to drop 2.2.x version from OrderResource_1_10 as a supported version and that should fix the issue without the order attribute. And if you think about, OrderResource_1_10 can't support 2.2.x since the new fullfiller details don't exist on it.